stylesheets/commons/Brackets.less

```diff
@@ -868,6 +868,11 @@
 	padding: 2px 1px 2px 1px;
 }
 
+.brkts-popup-side-color-yellow {
+	class: bg-first !important;
```

Comment:
```suggestion
	background-color: var( --achievement-placement-1, #ffd739 ) !important;
```

if you really want to reuse the var we use for placement 1
otherwise use a different css var (imo preferable)
e.g. from clr-sun

    --clr-sun-10: #332b00;
    --clr-sun-20: #665500;
    --clr-sun-30: #998000;
    --clr-sun-40: #ccab00;
    --clr-sun-80: #ffee99;
    --clr-sun-90: #fff7cc;

best to select one for light and one for darkmode